.buildkite/deploy.yml

```diff
@@ -0,0 +1,61 @@
+# Build and deploy to one of develop, staging, or production.
+#
+# Variables:
+#   VIRTRU_ENV: Environment to release to
+#   S3_PATH: path upload contents to on S3
+#   BLOCK_DEPLOY: [optional, defaults to true]
+
+env:
+  AWS_ACCOUNT_ALIAS: common
+  ENVIRONMENT_NAME: common
+  DEPLOY_ENVIRONMENT_NAME: production
+  S3_HOSTNAME: demos.developer.virtru.com
+  TEST_ENVIRONMENT_NAME: ${VIRTRU_ENV}
+  USE_NODE_12: true
+
+steps:
+  - label: 'annotate'
+    command: |
+      cat << EOF | buildkite-agent annotate --style "info"
+        [Triggered by ${BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG} Build #${BUILDKITE_TRIGGERED_FROM_BUILD_NUMBER}](/${BUILDKITE_ORGANIZATION_SLUG}/${BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG}/builds/${BUILDKITE_TRIGGERED_FROM_BUILD_NUMBER}#${BUILDKITE_TRIGGERED_FROM_BUILD_ID})
+      EOF
+    if: '"${BUILDKITE_SOURCE}" == "trigger_job"'
+
+  - label: ':house_with_garden: Create block for deploy environment selection'
```

Comment:
This makes sense for blocking deployments, but does this do anything or does it drive line48?

Reply:
this sets the `deploy-environment-name` meta-data object, which is used to fill the screaming snake cased env variable later in the pipeline

Reply:
🐍🐍🐍